mytoyota/controller.py

```diff
@@ -193,9 +193,18 @@ async def request(  # pylint: disable=too-many-branches
                 f"Body: {censor_dict(body) if body else body} - Parameters: {params}"
             )
             response = await client.request(
-                method, url, headers=headers, json=body, params=params
+                method,
+                url,
+                headers=headers,
+                json=body,
+                params=params,
+                follow_redirects=True,
             )
-            if response.status_code == HTTPStatus.OK:
+            if response.status_code in [
+                HTTPStatus.OK,
+                HTTPStatus.ACCEPTED,
```

Comment:
You are adding these to every request, so this will also affect all other request. Are you 100% sure this doesn't affect other endpoints?

Reply:
`OK` maps to a `200` HTTP status which is usually good when `GET`ing data, python maps `ACCPETED` to `202`, which means that the (lock/unlock) request has been accepted successfully. `FOUND` is a 302 which is a (temporary) redirect, also not an error status. It's just located at a different URI. Usually I would only throw if the code is `>=400`, but that's a personal choice

Reply:
BTW, the Toyota API seems to be quite inconsistent in style, for example the Set alias endpoint should also return a `202` among other discrepancies

Reply:
> BTW, the Toyota API seems to be quite inconsistent in style, for example the Set alias endpoint should also return a 202 among other discrepancies

Tell me about it 😆

We will let them be there, I don't think they are gonna cause problems for now. It just important that we tests this before a new release